src/processing/dom/index.js

```diff
@@ -216,6 +216,11 @@ export default class DomProcessor {
         this.adapter.setAttr(el, 'autocomplete', 'off');
     }
 
+    _processContentSecurityPolicy (el) {
+        if (this.adapter.getAttr(el, 'http-equiv').toLowerCase() === 'content-security-policy')
+            this.adapter.removeElement(el);
```

Comment:
Is it ok to remove element during traversing? Can it potentially brake anything? \cc @churkin